pyfar/classes/coordinates.py

```diff
@@ -2756,11 +2761,6 @@ def __getitem__(self, index):
 
         return new
 
```

Comment:
```suggestion
   def __array__(self, copy=True, dtype=None):
        """Instances of Coordinates behave like `numpy.ndarray`, array_like."""
        # copy to avoid changing the coordinate system of the original object
        return np.array(self.cartesian, copy=copy, dtype=dtype)


```